sdk/eventhub/testhub/package.json

```diff
@@ -32,7 +32,7 @@
     "unit-test": "npm run unit-test:node && npm run unit-test:browser"
   },
   "dependencies": {
-    "@azure/event-hubs": "^2.1.1",
+    "@azure/event-hubs": "5.0.0-preview.3",
```

Comment:
Why is this change being made? The testhub package has not been updated to use the latest version of event hubs yet

Reply:
Reverted in https://github.com/Azure/azure-sdk-for-js/pull/5324
It was required for a build fix, the common-version file changes from master fixes it